wikipedia-sentences-query-while-indexing/README.md

```diff
@@ -31,31 +31,31 @@ This is an example of using [Jina](http://www.jina.ai) to support both querying
 
 Querying while indexing means you are able to still query your data while new data is simultaneously being inserted (or updated, or deleted).
 Jina achieves this with its the dump-reload feature.
-For more in-depth technical information about how Jina achieves this, refer to [the docs](https://docs.jina.ai/chapters/dump-reload/)
 
 ## Configuration changes
 
 This feature requires you to split the Flow, one for Indexing (and Updates, Deletes) and one for Querying, and have them running at the same time.
 Also, you will need to replace the indexers in Flows.
-The Index Flow (also referred to as the DBMS Flow) will require a `DBMSIndexer`, while the Query Flow will require `QueryIndexer`.
-In our case we use `BinaryPbDBMSIndexer` and `CompoundQueryExecutor` (made up of `BinaryPbQueryIndexer` and `NumpyQueryIndexer`).
-These are the standard Indexers provided by Jina, but we also provide:
+The Index Flow (also referred to as the DBMS Flow) will require a [Storage Indexer](https://github.com/jina-ai/executors/tree/main/jinahub/indexers/storage), while the Query Flow requires a [Compound Searcher](https://github.com/jina-ai/executors/tree/main/jinahub/indexers/searcher).
```

Comment:
Replace reference to `DBMS Flow` with `Storage Flow`. `DBMS` is not used anywhere anymore, and has been deprecated